build/tools/CFamilyJarPreProcessor/CFamilyJarPreProcessor.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
New solution file referring to just the new tool project.
It's not strictly required, but it makes development and testing much quicker since I don't have to build the whole SLVS sln.